lib/player.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "coordinates.rb"
+
+class Player
+  attr_reader :interface, :coordinates
+
+  def initialize(interface:)
+    @interface = interface
+    @coordinates = get_move
```

Comment:
This happening in the initializer doesn't feel right to me. We have a method `make_move` to capture the input, but a first attempt at that will happen without calling the method. From the calling code, it means you can't instantiate the object until you're ready to ask for input.

Reply:
Agree!